data/presets/amenity/fast_food/cafeteria.json

```
@@ -0,0 +1,18 @@
+{
+    "icon": "maki-restaurant",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "eat",
+        "breakfast",
+        "dining",
+        "dinner"
+    ],
+    "tags": {
+        "amenity": "fast_food",
+        "fast_food": "cafeteria"
+    },
```

I'd include the `access` field as some cafeteria's might be not be open to the general public.

```suggestion
    },
    "fields": [
        "{amenity/fast_food}",
        "access_simple"
    ],
```
what do you think?

---

Yes, I agree